### DISCUSSION TITLE
지도상에서 시간문제

### DISCUSSION THREAD

**Discussion by hyojik123** [changdoc/naver-vaccine-macro#223] [Q&A]
예약페이지에서 중복 3페이지 정도 2초 새로고침잡아서  오류페이지 뜨는데 풀리고 다시 시도해볼려고 합니다.
지도상에서는 2초로 해도 막히나요? 
아니면 병원예약페이지에서 만 그런건가요? 
지도상에서도 5초대로 잡아야 할까요?

**Comment by changdoc** [comment]
예약 신청 페이지 차단은 예약 페이지에서 짧은 시간 내 새로고침, 다중 접속이 진행될 경우 발생하는 것으로 알고 있습니다.

지도에서의 동작은 짧은 시간으로 설정해둔 상태입니다.

**Reply by hyojik123** [reply]
지도에서는 짧은 시간해도 상관없나요? 
단점이 예약페이지로 들어가는 시간에 
다른 분이 먼저 잡을 수 있는거죵?

**Reply by changdoc** [reply]
네 지도는 짧은 시간에 새로고침해도 별도로 차단되는 상황은 없는 것으로 확인했습니다.

타이밍이 좀 늦을 수 있는 부분은 있긴합니다.
